src/components/Toolbar.vue

```diff
@@ -1,27 +1,27 @@
 <template lang="pug" xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
  div
-  v-toolbar(clipped-left fixed app)
-    v-toolbar-side-icon(@click.stop="drawer= !drawer")
+  v-app-bar(clipped-left fixed app)
+    v-app-bar-nav-icon(@click.stop="drawer= !drawer")
     v-toolbar-title {{ title }}
     v-spacer
     v-toolbar-items(hidden-sm-and-down)
-      v-btn(flat)
+      v-btn(text)
         v-icon(left) mdi-forum
         span(v-if="$vuetify.breakpoint.mdAndUp") Forum
-      v-btn(flat href="../konnekt")
+      v-btn(text href="../konnekt")
```

Comment:
Avoid relative hrefs

Reply:
What should I do then?

Reply:
@ajatprabha should I always use absolute href?

Reply:
@ajatprabha when should I use absolute href and when should I use relative href's?

Reply:
Both of them should be avoided. Always use programmatic links through the router.
Ex: instead of `v-btn(flat href="../konnekt")`, use `v-btn(flat :to="{name: 'konnekt-home'}")`
This way you can change the path later in the `routes.js` and you won't have to update any code inside of a component.